.github/workflows/publish.yml

```diff
@@ -0,0 +1,110 @@
+name: Publish
+
+on:
+  push:
```

Comment:
Is this action triggered by push on `master` branch only?

Reply:
IIUC, it's triggered by push of tags only.

Reply:
The pushed tag has to match `v[0-9]+.[0-9]+.[0-9]+`

This does mean that someone could trigger a release from a 'feature' branch, I can see arguments either way for why that may or may not be ideal.

Personally I think it's fine as is, as long as everyone understands that pushing a tag of that form will trigger a release workflow. Though if we think it's safer I can add an additional check to ensure it only runs on tag pushes on `master` (or any pattern we think is sensible)